src/Exception/InvalidCallbackReturnTypeException.php

```diff
@@ -4,28 +4,50 @@
 
 namespace Yiisoft\Validator\Exception;
 
+use Exception;
 use Throwable;
 use Yiisoft\FriendlyException\FriendlyExceptionInterface;
 use Yiisoft\Validator\Result;
+use Yiisoft\Validator\Rule\Callback;
+use Yiisoft\Validator\Rule\CallbackHandler;
 
-final class InvalidCallbackReturnTypeException extends \Exception implements FriendlyExceptionInterface
+/**
+ * An exception used by the handler ({@see CallbackHandler}) of {@see Callback} rule for the cases when returned value
+ * is not a {@see Result} instance.
+ */
+final class InvalidCallbackReturnTypeException extends Exception implements FriendlyExceptionInterface
 {
-    public function __construct(mixed $result, int $code = 0, ?Throwable $previous = null)
```

Comment:
Why removed code?

Reply:
For consistency with other exceptions. I mentioned it in the PR's description too.